dashboard/test/ui/runner.rb

```diff
@@ -51,18 +51,22 @@
   opts.on("-f", "--feature Feature", Array, "Single feature or comma separated list of features to run") do |f|
     $options.feature = f
   end
-  opts.on("-p", "--pegasus Domain", String, "Specify an override domain for code.org, e.g. localhost:9393") do |d|
-    $options.pegasus_domain = d
+  opts.on("-l", "--local", "Use local webdriver, not Saucelabs") do
+    $options.local = 'true'
+    $options.pegasus_domain = 'localhost.code.org:3000'
+    $options.dashboard_domain = 'localhost.studio.code.org:3000'
   end
-  opts.on("-d", "--dashboard Domain", String, "Specify an override domain for studio.code.org, e.g. localhost:3000") do |d|
+  opts.on("-p", "--pegasus Domain", String, "Specify an override domain for code.org, e.g. localhost.code.org:3000") do |p|
+    raise "'localhost:3000' is an invalid pegasus domain. Try '-p localhost.code.org:3000'." if p == 'localhost:3000'
+    $options.pegasus_domain = p
+  end
+  opts.on("-d", "--dashboard Domain", String, "Specify an override domain for studio.code.org, e.g. localhost.studio.code.org:3000") do |d|
+    raise "'localhost:3000' is an invalid dashboard domain. Try '-d localhost.studio.code.org:3000'." if d == 'localhost:3000'
```

Comment:
Sounds good, especially since most of the time people will now just be running -l and get the localhost.studio.code.org without needing to specify -d